src/rftg.h

```diff
@@ -61,17 +62,17 @@
 /*
  * Maximum number of expansion levels.
  */
-#define MAX_EXPANSION 5
+#define MAX_EXPANSION 6
 
 /*
  * Number of available card designs slots.
  */
-#define AVAILABLE_DESIGN 256
+#define AVAILABLE_DESIGN 280
```

Comment:
It came from popular request where people wanted to add their own cards to the game by
1. putting more card definitions to cards.txt (only using existing powers of course)
2. adding custom images of the cards as separate .jpg files.

Of course the AI would be completely oblivious about the new cards, but it was a simple change that made some people happy.